server/game/core/event/EventWindow.js

```diff
@@ -21,6 +21,8 @@ class EventWindow extends BaseStepWithPipeline {
 
         this.triggeredAbilityWindow = null;
 
+        this.resolveTriggersAfter = resolveTriggersAfter;
```

Comment:
I can't think of anything great. Does "resolveTriggersAfterEvents" disambiguate at all?

Reply:
I don't think so - maybe a different direction, like "hasTriggeringWindow" since in some cases it technically doesn't have a triggering window? Or like "parentWindowResolvesTrigger" or something

Reply:
How about ownsATriggerWindow?

Reply:
Maybe hasTriggerWindow or hasTriggeredAbilityWindow (since "trigger" could mean a lot of things)? "Owns" feels a little weird in this context

Reply:
Well all EventWindows will have a TriggeredAbilityWindow. It's just a question of whether that window is shared with other EventWindows and which EventWindow will cause it to resolve.